src/styles/_card.scss

```diff
@@ -0,0 +1,123 @@
+@use './variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: variables.$width-card;
+  background-color: variables.$white-color;
+  padding: variables.$padding-card;
+  border: 1px solid variables.$crem-color;
+  margin: variables.$margin-card;
+  transition: transform 0.3s ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-weight: variables.$title-font-weight;
+    font-size: variables.$title-font-size;
+    color: variables.$main-color;
+    line-height: 18px;
+    margin: 0 0 4px;
+    transition: color 0.3s ease;
+
+    .card:hover & {
+      color: variables.$dark-blue-color;
```

Comment:
The hover effect for the card title should change the color to `#34568b` as specified in the task requirements. Ensure that the color variable used here matches this requirement.